gnocchi/rest/api.py

```diff
@@ -1677,25 +1732,8 @@ def get_cross_metric_measures_from_objs(metrics, start=None, stop=None,
                                             granularity=None,
                                             needed_overlap=100.0, fill=None,
                                             refresh=False, resample=None):
-        try:
-            needed_overlap = float(needed_overlap)
-        except ValueError:
-            abort(400, 'needed_overlap must be a number')
-        if needed_overlap != 100.0 and start is None and stop is None:
-            abort(400, 'start and/or stop must be provided if specifying '
-                  'needed_overlap')
-
-        if start is not None:
-            try:
-                start = utils.to_timestamp(start)
-            except Exception:
-                abort(400, "Invalid value for start")
-
-        if stop is not None:
-            try:
-                stop = utils.to_timestamp(stop)
-            except Exception:
-                abort(400, "Invalid value for stop")
+        start, stop, granularity, needed_overlap, fill = validate_qs(
```

Comment:
this will change the response error. I'm ok with that I think but I want to say it here so @chungg can convince me it's a bad idea

Reply:
i'm indifferent. i imagine we could hide it in client if we really cared...